source/command-logging-and-monitoring/tests/monitoring/unacknowledged-client-bulkWrite.yml

```
@@ -62,9 +66,21 @@ tests:
             $$unsetOrMatches: {}
           deleteResults:
             $$unsetOrMatches: {}
+      # Force completion of the w:0 write by executing a find on the same connection
+      - object: *collection
+        name: find
+        arguments:
+          filter: {}
+        expectResult:
+          - { _id: 1 }
+          - { _id: 2 }
+          - { _id: 3 }
+          - { _id: 4 }
```

```suggestion
          - { _id: 1, x: 11 }
          - { _id: 2, x: 22 }
          - { _id: 3, x: 333 }
          - { _id: 4, x: 44 }
```

I'm not sure if you were relying on ignoring extra fields in root level documents (per [Evaluating Matches](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.md#evaluating-matches)), but the [original tests](https://github.com/mongodb/specifications/blob/master/source/command-logging-and-monitoring/tests/monitoring/unacknowledgedBulkWrite.yml) did match the full document.